npm/iptm/iptm.go

```diff
@@ -221,6 +212,7 @@ func (iptMgr *IptablesManager) checkAndAddForwardChain() error {
 	if !exists {
 		// position Azure-NPM chain after Kube-Forward and Kube-Service chains if it exists
 		iptMgr.OperationFlag = util.IptablesInsertionFlag
+		index := 1
 		entry.Specs = append([]string{strconv.Itoa(index)}, entry.Specs...)
```

Comment:
is `index` never not `1`? seems like we could save the strconv on the next line. maybe also the inline array initialization

Reply:
the initial commits here were just to validate something. now we use a toggle so index is variable